lib/utils/generate-name.js

```diff
@@ -0,0 +1,20 @@
+const Moniker = require("moniker");
+
+/**
+ * Generates a random team name in the form 'AdjectiveNoun'
+ */
+const generateName = () => {
+  const names = Moniker.generator([Moniker.adjective, Moniker.noun], {
```

Comment:
is a small thing but might be more performant to create the name generator only once as a `const` outside the function?

Also, we've shifted code convention,  so can we make this `function generateName() {`?